main/solution/edge-lambda/config/settings/.defaults.yml

```diff
@@ -34,7 +34,7 @@ apiUrl: ${cf:${self:custom.settings.backendStackName}.ServiceEndpoint}
 #   This is required to auto-populate user's IP address when launching EC2 based workspace from the UI
 #
 #otherConnectSrc: cloudformation.us-east-2.amazonaws.com cloudformation-fips.us-east-2.amazonaws.com cloudformation.us-east-1.amazonaws.com cloudformation-fips.us-east-1.amazonaws.com cloudformation.us-west-1.amazonaws.com cloudformation-fips.us-west-1.amazonaws.com cloudformation.us-west-2.amazonaws.com cloudformation-fips.us-west-2.amazonaws.com cloudformation.ap-south-1.amazonaws.com cloudformation.ap-northeast-3.amazonaws.com cloudformation.ap-northeast-2.amazonaws.com cloudformation.ap-southeast-1.amazonaws.com cloudformation.ap-southeast-2.amazonaws.com cloudformation.ap-northeast-1.amazonaws.com cloudformation.ca-central-1.amazonaws.com cloudformation.cn-north-1.amazonaws.com.cn cloudformation.cn-northwest-1.amazonaws.com.cn cloudformation.eu-central-1.amazonaws.com cloudformation.eu-west-1.amazonaws.com cloudformation.eu-west-2.amazonaws.com cloudformation.eu-west-3.amazonaws.com cloudformation.sa-east-1.amazonaws.com cloudformation.sa-east-1.amazonaws.com cloudformation.us-gov-west-1.amazonaws.com
-otherConnectSrc: '*.amazonaws.com'
+otherConnectSrc: '*.amazonaws.com *.cloudfront.net'
```

Comment:
@dengmingtong any comment for this change?

Reply:
Not sure if @dengmingtong is a typo, but from my point of view I can explain why I added this change. According to the LogHub file structure, browsers now request language resource files from the website URL instead of the API URL, which is not included in the content-security-policy, so the resource request fails and the home page cannot be accessed.

Reply:
I did not modify this files, it is not a typo. 
For China regions, we skipped lambda edge. I think this config is for global regions, right?
Please think about the case that customer configure the domain, not use cloudfront.net directly.
There is a config item: domainName.

Reply:
> I did not modify this files, it is not a typo. For China regions, we skipped lambda edge. I think this config is for global regions, right? Please think about the case that customer configure the domain, not use cloudfront.net directly. There is a config item: domainName.

These is **NO** `domainName` in `main/solution/edge-lambda/config/settings/.defaults.yml`，replacing `*.cloudfront.net ` with `${self:custom.settings.domainName}` will only recreate the CSP violation problem.

<img width="1451" alt="image" src="https://user-images.githubusercontent.com/106293323/179898549-b1c7cdde-f889-4dfb-8a84-db6898bfa505.png">

Reply:
domainName is not a mandatory config, I think you need keep *.cloudfront.net, and also add ${self:custom.settings.domainName}. 
domainName is in <stage>.yml if customer configured

Reply:
Both kept in the latest commit